### PR TITLE
Upgrade robolectric to latest version.

### PR DESCRIPTION
**Pull request by vkryachko**

### CODE REVIEW

File: firebase-inappmessaging-display/firebase-inappmessaging-display.gradle [hunk 1]
```diff
@@ -15,11 +15,11 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
```
Comment (mikelehen) [inline firebase-inappmessaging-display/firebase-inappmessaging-display.gradle]: Just to confirm, is this change (and below) intentional?

Comment (vkryachko) [reply firebase-inappmessaging-display/firebase-inappmessaging-display.gradle]: Yep, robolectric 4+ requires minimum compileSdkVersion = 28

**Pull request merged**